ibexa/commerce/4.0.x-dev/encore/ibexa.webpack.config.js

```diff
@@ -24,20 +23,20 @@ module.exports = (Encore) => {
         addEntries(Encore);
     });
 
-    const Config = Encore.getWebpackConfig();
+    const ibexaConfig = Encore.getWebpackConfig();
 
-    Config.name = 'ibexa';
+    ibexaConfig.name = 'ibexa';
 
-    Config.module.rules[4].oneOf[1].use[1].options.url = false;
-    Config.module.rules[1].oneOf[1].use[1].options.url = false;
+    ibexaConfig.module.rules[4].oneOf[1].use[1].options.url = false;
+    ibexaConfig.module.rules[1].oneOf[1].use[1].options.url = false;
 
     configManagers.forEach((configManagerPath) => {
         const configManager = require(configManagerPath);
 
-        configManager(Config, ConfigManager);
+        configManager(ibexaConfig, ibexaConfigManager);
```

Comment:
Question: previous convention with uppercase of the first letter was wrong?
```suggestion
        configManager(IbexaConfig, IbexaConfigManager);
```

Reply:
Discussed in private: it seems there was no reason before to use uppercase.